src/client/yoti.client.js

```diff
@@ -29,13 +29,23 @@ class YotiClient {
   /**
    * @param {string} sdkId
    * @param {string} pem
+   * @param {Object} options
+   * @param {string} options.apiUrl
    */
-  constructor(sdkId, pem) {
+  constructor(sdkId, pem, { apiUrl } = {}) {
     this.sdkId = sdkId;
     this.pem = pem;
 
+    const options = {
+      apiUrl: apiUrl || config.yoti.connectApi,
```

Comment:
Minor: I see that the apiUrl here is being decided on via some logic inside of the `options`. For consistency should we keep that logic inside of the constructor destructuring like how it is used in other places:

aml_service usage: `{ apiUrl = DEFAULT_API_URL } = {}`

Reply:
Yes, good point!

The rational for having a different way of doing is not very strong, mainly about saying that `services` options are directly consumed and store within the class instances, therefore must be passed as valid or defaulted in case of.
On the other hand, the `clients` are instantiating their services and "need" to assemble an options object (which in the case of the `YotiClient` is shared), so it seemed more appropriate to clearly state that preparation phase as a separate block (which you can also then share across the different service) and also could allow more logic to be done when defaulting some values...
As mentioned, not very strong argument... but I guess I went this way as in the end it helps dissociating the `clients` and `services` too.